### PR TITLE
feed and post visibility toggle

### PR DESCRIPTION
**Pull request by alternadiva**

### CODE REVIEW

File: components/journal/JournalFeed.js [hunk 1]
```diff
@@ -13,13 +14,17 @@ function JournalFeed(props) {
...
         const href = `/posts/${art.id}`;
         return (
-          <li key={art.id}>
-            <h3>{art.mood}</h3>
-            <p>{art.date}</p>
-            <ArtUploadImage src={art.img} alt={art.alt} />
-            <p>{art.caption}</p>
-            <button onClick={() => router.push(href)}>Open...</button>
-          </li>
+          <ListItem
```
Comment (PJSalter) [inline components/journal/JournalFeed.js:17]: Great work on the list items ✨

File: components/journal/ListItem.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import { useRouter } from "next/router";
+import DeletePost from "./DeletePost";
+import UpdatePost from "./UpdatePost";
+
+function ListItem(props) {
+  const [visibility, setVisibility] = useState(false);
+
+  const router = useRouter();
+
+  const date = new Date(props.inserted_at).toLocaleString();
```
Comment (PJSalter) [inline components/journal/ListItem.js:11]: So does this set a new date each time? awesome 🎨

File: components/journal/UpdatePost.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        public: checkedVisibility,
+      })
+      .match({ id: props.id });
+    setFormVisibility(!formVisibility);
+
+    router.push("/journal");
+  }
+
+  return (
+    <div>
```
Comment (PJSalter) [inline components/journal/UpdatePost.js:39]: Very clear to read on form elements ✨

**Review by PJSalter — APPROVED** [review]

File: components/art-posts/ListItem.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      <h3>Mood: {props.mood}</h3>
+      <p>{date.slice(0, 10)}</p>
+      <button onClick={toggleVisibility}>Hide post</button>
+      <img src={props.img} alt={props.alt} />
+      <p>Username: {props.email}</p>
+      <p>{props.caption}</p>
+      <button onClick={() => router.push(props.href)}>Open...</button>
+    </li>
+  ) : (
+    <button onClick={toggleVisibility}>Show post again</button>
```
Comment (AsmahanM) [inline components/art-posts/ListItem.js:24]: Really like this feature! Nice work

**Review by AsmahanM — APPROVED** [review]

**Pull request merged**